litmus-portal/frontend/src/components/VideoFrame/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import useStyles from './styles';
+
+interface IVideoBox {
+  width: string;
+}
+interface IVideoFrame extends IVideoBox {
+  src: string;
+}
+
+const VideoFrame: React.FC<IVideoFrame> = ({ width, src }) => {
+  const classes = useStyles();
+  return (
+    <div className={classes.videoDiv}>
+      <iframe
+        title="chaos-video"
+        src={src}
+        frameBorder="0"
+        allow="accelerometer"
+        allowFullScreen
+        className={classes.videoBox}
+        style={{ width, height: `calc(${width} / (16 / 9))` }}
```

Comment:
Please try to include this into the 'styles.ts' file as well, using ',' the CSS selector.

Reply:
^^ It may be so that this is not possible/done in a different way, but I think I've seen @arkajyotiMukherjee, do something similar, please advice.

Reply:
@A4k9  I think we did it purposely to have a 16:9 ratio for the video frame.